src/main/java/org/opentripplanner/client/OtpApiClient.java

```diff
@@ -113,6 +114,27 @@ public List<Pattern> patterns() throws IOException {
     return deserializeList(json, type, "/data/patterns");
   }
 
+  /**
+   * Returns a TripPlan, also known as a routing result.
+   *
+   * @link <a href="https://docs.opentripplanner.org/api/dev-2.x/graphql-gtfs/queries/plan">OTP API
+   *     docs</a>
+   */
```

Comment:
Please update the documentation.

Reply:
Classic Copy-Paste-Error. It is fixed now.